frontend/pages/queries/ManageQueriesPage/ManageQueriesPage.tsx

```diff
@@ -128,9 +133,16 @@ const ManageQueriesPage = ({
       refetchOnWindowFocus: false,
       enabled: isRouteOk,
       select: (data) => data.queries.map(enhanceQuery),
+      onSuccess: (data) => {
+        const scheduledQueries = data
+          .filter((query) => query.automations_enabled)
+          .map((query) => query.id);
+        setAutomatedQueryIds(scheduledQueries);
```

Comment:
Since useQuery will be keeping track of this state, it might be less redundant to have a memoized value that watches this directly instead of having another state variable. Something like:

```
const automatedQueryIds = useMemo(curTeamEnhancedQueries.filter((query) => query.automations_enabled)
          .map((query) => query.id), [curTeamEnhancedQueries]);
```
and then we can remove line 122. For a future iteration perhaps.

Reply:
Done!